buildspec.yml

```diff
@@ -19,5 +19,6 @@ artifacts:
     - output/images/ot2-system.zip
     - output/images/ot2-fullimage.zip
     - output/images/VERSION.json
+    - output/images/release-notes.md
```

Comment:
Will the URI for this release-notes file also be put in the top level `releases.json`?

Reply:
@mcous yes, by https://github.com/Opentrons/devops/pull/10 - the webhook handles that